packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -205,9 +205,8 @@ class PersonalDetailsForm extends React.Component {
 
         const state_is_input_element = values.address_state && !this.props.states_list.length;
         if (state_is_input_element) {
-            const max_state_length = 35;
-            if (!validLength(values.address_state, { min: 0, max: max_state_length })) {
-                errors.address_state = localize('You should enter 0-35 characters.');
+            if (!/^[\w\s\W'.-;,]{0,60}$/.test(values.address_state)) {
+                errors.address_state = localize('State is not in a proper format');
```

Comment:
usually we check length and format in two different lines so we can show a more useful error message, one about length and one about which characters are allowed, why are we changing this now?

Reply:
There is an inconsistency with the previous rule and the new rule (in signup and account). I'll bring back the length check.